en/hardware/pwm_escs_and_servo.md

```diff
@@ -0,0 +1,132 @@
+A higher update rate is then desirable to effectively limit the dead time between control output and motor update. 
+In any of these cases the main limiting factor is the PWM signal spec itself, which takes 1-2 ms (0-100% motor speed) to transmit and limits the update rate to < 500 Hz.
+
+## Connecting the Ground Wire
+
+An electric signal is defined as the deviation from a nominal zero-value. 
+For normal signals this reference is ground (GND). 
+Therefore just one wire is never enough to transmit a signal - it always needs a reference (on differential signals, the reference is established by the difference). 
+A wiring setup not in need of a ground reference doesn't exist, and if the path the current has to take to "return" is much longer than the signal wire, complex electromagnetic effects can come into play. 
+The image below shows how noisy the signal becomes if GND is not connected.
+
+![PWM without ground](../../assets/hardware/pwm_esc/pwm_without_gnd.png)
+
+
+## Signal Shape Comparison
+
+As the scope shots below show the Pixhawk signal arrives faster (at 400 Hz when configured for multicopters instead of 50-60 Hz), but the overall signal shape and voltage levels are exactly the same as for common RC receivers.
+
+### FrSky D4R-II 
+
+3.3V levels, 55 Hz (default, users are advised to flash the 27 ms / 37 Hz firmware)
+
+![frsky_d4_rii](../../assets/hardware/pwm_esc/frsky_d4_rii.png)
+
+
+### Futaba R7008SB
+
+3.0V levels, 66 Hz
+
+![futaba_r7008sb](../../assets/hardware/pwm_esc/futaba_r7008sb.png)
+
+### Spektrum AR 10000
+
+3.3V levels, 45 Hz
+
+
+![spektrum_ar_10000](../../assets/hardware/pwm_esc/spektrum_ar_10000.png)
+
+### Pixhawk (PX4FMU v2.4.3)
+
+3.3V levels, 400 Hz (**400 Hz ONLY if configured as multicopter, default is 50 Hz**)
+
+![pixhawk_400hz](../../assets/hardware/pwm_esc/pixhawk_400hz.png)
+
+## Troubleshooting
+
+Pixhawk is compatible with all ESCs on the market (as the comparison with the common RC receiver signals showed). 
+If a particular ESC is not operational, it is either wired up incorrectly, or the flight control stack used sends out the wrong values for this particular ESC.
+
+### Ground Connection
+
+Check that the ground (black wire) of the ESC servo connector is connected to Pixhawk. 
+**IT IS UNSAFE TO FLY WITHOUT GROUND CONNECTED**. 
+This is because for every positive pulse (the ESC signal) there needs to be an adjacent ground return path for a clean signal shape.
+
+### Power Connection / Opto-isolated ESCs
+
+Please see the first section of this page for a complete explanation on power connection considerations. 
+If you are using an opto-isolated ESC not providing a BEC / power output, please ensure that the ESC does not need its +5V line powered for the opto-isolator.
+
+### Missing Valid Minimum Value
+
+If the ESC requires to see a special low value pulse first before switching on, it is necessary that the flight control stack sends this value initially. 
```

Comment:
This is a good point and I had cases where the ESC would not initialize until I slightly changed the PWM_DISARMED value.